lib/modules/userInfo.js

```diff
@@ -209,6 +212,26 @@ async function showUserInfo(card) {
 		userHTML += string.escape`<div class="${isHighlight ? 'blueButton' : 'redButton'}" id="highlightUser" data-userid="${data.id}">${isHighlight ? i18n('userInfoHighlight') : i18n('userInfoUnhighlight')}</div>`;
 	}
 
+	if (userTag) {
+		userHTML += string.escape`<a target="_blank" rel="noopener noreferer" id="blockUser" class="blueButton">${i18n('userInfoBlock')}</a>`;
+
+		$body.on('click', '#blockUser', () => {
+			loggedInUserHash().then(
+				hash => {
+					ajax({
+						method: 'POST',
+						url: '/api/block_user',
+						data: {
+							account_id: `t2_${userTag.id}`,
+							api_type: 'json',
+							name: userTag.id,
+							uh: String(hash) },
+					}).then(() => FilteReddit.listFilters.users.toggleString(userTag.id, true, false));
```

Comment:
This may be inadvertently be saved at a later stage, e.g. if another user is ignored. It might be simpler to emulate the blocking behavior by adding a CSS snippet:

```addCSS(`.thing[data-author="${data.name}"] { display: none !important; }`)```